src/html/generator.ml

```diff
@@ -43,7 +43,7 @@ let mk_anchor anchor =
 let class_ (l : Class.t) = if l = [] then [] else [ Html.a_class l ]
 
 let math (s : Math.t) =
-  Html.span ~a:[ Html.a_class [ "odoc-katex-math" ] ] [ Html.txt s ]
+  Html.span ~a:[ Html.a_class [ "odoc-katex-math"; "display" ] ] [ Html.txt s ]
```

Comment:
The `display` class should only be added in blocks - i.e. in the test, the one that says it's inline should be `class="odoc-katex-math"` and the one that says it's a block should be `class="odoc-katex-math display"`

Reply:
I should have been more explicit in my previous comment, sorry! :-)

Reply:
Done, but some fonts are not properly rendered even though every font file has been vendored and is correctly located in the test (don't have any log error).